html/modules/custom/reliefweb_import/src/Service/InoreaderService.php

```diff
@@ -426,7 +426,15 @@ public function processDocumentData(array $document): array {
             break;
 
           case 'content':
-            $body = $this->cleanBodyText($document['summary']['content'] ?? '');
+            $body = $document['summary']['content'] ?? '';
+
+            // Remove specified HTML elements.
+            if (isset($tags['remove'])) {
+              $body = $this->removeHtmlElements($body, $tags['remove']);
+            }
+
+            $clean_body = ($tags['content'] == 'clean') ?? FALSE;
```

Comment:
```suggestion
            $clean_body = isset($tags['content']) && $tags['content'] == 'clean';
```

`($tags['content'] == 'clean') ?? FALSE;` results in a PHP warning undefined property if `content` is not present.